backend/buildeR.API/Controllers/AuthController.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using buildeR.Common.DTO.User;
+using buildeR.Common.Enums;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace buildeR.API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class AuthController : ControllerBase
+    {
+        [HttpGet]
+        public IActionResult GetUserFromToken(string token)
```

Comment:
Do not use IActionResult, make actual dto type. Token is going to be in headers. Don't need to pass as a parameter.